src/main/java/mekanism/client/recipe_viewer/jei/ChemicalStackHelper.java

```diff
@@ -41,9 +43,10 @@ public String getDisplayName(ChemicalStack ingredient) {
     }
 
     @Override
-    @Deprecated(forRemoval = true)
+    @SuppressWarnings("removal")
+    @Deprecated(forRemoval = true, since = "19.9.0")
```

Comment:
19?

Reply:
That's the JEI version that it was deprecated for removal in.

Reply:
ahh, perhaps we can add `JEI: ` to the string?